src/types/predicates.go

```diff
@@ -37,6 +37,23 @@ func IsAlias(s string) bool {
 	return has
 }
 
+func IsTypeNullable(typ Map) bool {
+	isNullable := false
+	if typ.m == nil {
+		return true // We consider that if the type is not inferred, then it is mixed
+	}
+	typ.Iterate(func(t string) {
+		if t == "null" || strings.Contains(t, "mixed") {
```

Comment:
`t.Contains("mixed")` || `t.Contains("null")`

Reply:
why not:

```
func IsTypeNullable(typ Map) bool {
	return typ.Contains("null") || typ.Contains("mixed")
}
```

Reply:
Ok, but this should be: 
```
	if typ.m == nil {
		return true // We consider that if the type is not inferred, then it is mixed
	}
```